auth_proxy/proxy/flask.py

```diff
@@ -99,3 +105,32 @@ def _patient_security(self):
             return 'Patient/{}'.format(patient)
         except AttributeError:
             return 'public'
+
+
+class OpenFlaskClient(FlaskClient):
+    """Subclass of FlaskClient, overridden to prevent any security checks """
+    def __init__(self, url, orig):
```

Comment:
If you don't override init, it'll just work, right?

Reply:
Yep!